1-js/02-first-steps/16-javascript-specials/article.md

```diff
@@ -1,180 +1,180 @@
-# JavaScript specials
+# Especiais em JavaScript
 
-This chapter briefly recaps the features of JavaScript that we've learned by now, paying special attention to subtle moments.
+Este capítulo brevemente revê as funcionalidades de JavaScript que aprendemos até agora, dando atenção especial a momentos subtis.
 
-## Code structure
+## Estrutura do código
 
-Statements are delimited with a semicolon:
+Instruções são delimitadas por ponto-e-vírgula:
```

Comment:
Since the purpose is to teach, clarity is sought here.

Furthermore, here is a link to a Portuguese site:
https://ciberduvidas.iscte-iul.pt/consultorio/perguntas/ponto-e-virgula/127https://ciberduvidas.iscte-iul.pt/consultorio/perguntas/ponto-e-virgula/127